src/test/java/api/support/fakes/StorageSchema.java

```diff
@@ -30,4 +30,8 @@ public static JsonSchemaValidator validatorForLocationLibSchema() throws IOExcep
   public static JsonSchemaValidator validatorForCheckInStorageSchema() throws IOException {
     return JsonSchemaValidator.fromResource("/check-in-storage-0-2.json");
   }
+
+  public static JsonSchemaValidator validatorForNoteSchema() throws IOException {
+    return JsonSchemaValidator.fromResource("/note.json");
```

Comment:
Please could the file name include the version of the interface this is based upon, 2.9 I think?